model/src/main/proto/oppia_logger.proto

```diff
@@ -368,6 +375,30 @@ message EventLog {
     string profile_id = 2;
   }
 
+  // Structure of a feature flag context.
+  message FeatureFlagContext {
+    // The unique UUID of the user.
+    string unique_user_uuid = 1;
```

Comment:
I think this ID needs to actually be logged for all events, otherwise we won't be able to associate feature flag states with individual analytics (which prevents the goal of performing A/B comparison of app behaviors based on feature flags).

Reply:
The plan is to eventually have all logs include a uuid. This will be done once the UUID has been implemented as part of the Core Metrics implementation as detailed in [the TDD](https://docs.google.com/document/d/11aL2xt4aFv8ZyW9uu5QbP3hZ1XqYuLWvtZu1NuKkAxs/edit#heading=h.p47sq11zcyy1).

Reply:
Got it--thanks for clarifying.